docs/gathered-data.md

```diff
@@ -69,6 +69,34 @@ None
 None
 
 
+## AggregatedInstances
+
+Collects instances of `Prometheus` and `AlertManager` deployments
+that are outside of the `openshift-monitoring` namespace
```

Comment:
I think it would be good to describe it as custom resources and perhaps mention the full group and kind